### PR TITLE
Feat add dynamic render for graphs

### PR DESCRIPTION
**Pull request by dkryaklin**
feat: changed event type for range input from "change" => "input" to render graphs in real-time
https://github.com/evilmartians/oklch-picker/pull/43/files#diff-4384301177f20f8caa9e8babc120a27cc0a62c5ae643964923b550278db1222aR31

feat: added postponed frames feature that allows skipping renders to increase performance
https://github.com/evilmartians/oklch-picker/pull/43/files#diff-9a0117fd6cf1e041c4cb78e8dfd35d3058bd259fd47f2abb66435e3bc78d0e88R112-R125

feat: added ability to render low-quality frames to improve intermediate render performance
feat: added dynamic scale calculations to improve intermediate render performance
https://github.com/evilmartians/oklch-picker/pull/43/files#diff-d4b214cdd799ad8e568b91d6dcb24d1dd94541bbbf7f4db9f671863bcc9b45eeR49-R70

fix: add debounce for change hash in URL since there is a bug in safari that fail the render
https://github.com/evilmartians/oklch-picker/pull/43/files#diff-9a0117fd6cf1e041c4cb78e8dfd35d3058bd259fd47f2abb66435e3bc78d0e88R56-R62

https://user-images.githubusercontent.com/2098350/170865284-7c3d05f4-4d64-48f3-9dc2-5c7dec4c9147.mp4

### CODE REVIEW

File: view/range/index.ts [hunk 1]
```diff
@@ -26,8 +28,14 @@ function initRange(
...
   let range = div.querySelector<HTMLInputElement>('.range_input')!
 
-  range.addEventListener('change', () => {
-    current.setKey(type, parseFloat(range.value))
+  range.addEventListener('input', () => {
+    let renderingValue = isRendering.get()
+
+    postponed.set({ ...current.get(), [type]: parseFloat(range.value) })
+
+    if (!renderingValue[type]) {
```
Comment (ai) [inline view/range/index.ts:36]: Can we hide this logic in the store to have more high-level API in `view/`.

The idea of Nano Stores to hide as much as possible in stores.

**Comment by ai** [comment]
Wow! Amazing job!

Can I ask you to update `.size-limit.json` and resolve conflict?

**Comment by dkryaklin** [comment]
> Wow! Amazing job!
> 
> Can I ask you to update `.size-limit.json` and resolve conflict?

Done!

File: view/range/index.ts (hunk 1 above)
Comment (dkryaklin) [reply view/range/index.ts]: Yes, that is a good idea

**Comment by ai** [comment]
Works perfectly in Firefox.

Do you know why are starting to have a slower FPS in Chrome a 1-2 seconds after the start?

https://user-images.githubusercontent.com/19343/170900856-b92feeda-8293-41aa-a21e-5a1803eec94f.mp4

**Comment by ai** [comment]
BTW, we can always have a constant size of preview chart if it will fix the problem in Chrome

**Comment by ai** [comment]
> BTW, do you want to keep web worker implementation? Since render works much faster we can remove that and decrease bundle size and code complexity

Good idea. Do you want to do it in this PR?

**Comment by dkryaklin** [comment]
1. Separate PR is better in my opinion
2. There is a default value for fast-render, so the initial render is also faster now. As I can see there is no freeze, at least much less than 200ms

I just found a bug: after changing opacity it does not update the hash in the URL. I will fix that now and we are good to merge I think

**Comment by dkryaklin** [comment]
Pushed last changes

**Comment by ai** [comment]
Great work! Thanks

**Comment by ai** [comment]
I will remove worker on my own tomorrow

**Pull request merged**